gene_finder.py

```diff
@@ -62,8 +75,15 @@ def rest_of_ORF(dna):
     >>> rest_of_ORF("ATGAGATAGG")
     'ATGAGA'
     """
-    # TODO: implement this
-    pass
+    stop_codon_list = ['TAG', 'TAA', 'TGA']
+    orf = ''
+    for i in range(0, len(dna), 3):
+        codon = dna[i:i+3]
+        if codon in stop_codon_list:
+            return orf
+        else:
+            orf = orf + codon
+    return dna[:]
```

Comment:
`dna` means the same thing as `dna[:]`. Here you're essentially taking a slice of the whole string, when you could just return the string. (This doesn't change the meaning of your code at all, however.)